### PR TITLE
Respa admin: Add PriceListCopyView (TTVA-98)

### PR DESCRIPTION
**Pull request by jopesy**
Add a view for copying a price list in Respa Admin.

image

Refs TTVA-98

### CODE REVIEW

File: respa_admin/views/prices.py [hunk 1]
```diff
@@ -224,3 +229,28 @@ class PriceListDeleteView(ExtraContextMixin, DeleteView):
     def get_queryset(self):
         qs = super().get_queryset()
         return qs.modifiable_by(self.request.user)
+
+
+class PriceListCopyView(PriceListCreateView):
+    def get(self, request, *args, **kwargs):
+        self.object = None
+        id = self.kwargs.get("price_list_id")
```
Comment (danjacob-anders) [inline respa_admin/views/prices.py]: Minor point: should be e.g. `original_price_list_id` or something as `id` is a reserved Python keyword

Comment (jopesy) [reply respa_admin/views/prices.py]: Yeah, good point. I don't even need to use a variable here.

Comment (jopesy) [reply respa_admin/views/prices.py]: Fixed

**Review by danjacob-anders — APPROVED** [review]
Looks great!

**Pull request merged**